### PR TITLE
update to AS v0.2x.x

### PR DESCRIPTION
**Pull request by roaminro**
## Brief description
AS v0.20 introduces breaking changes, this PR updates everything needed in order to be able to use AS v0.2x.x

## Checklist

- [ x I have built this pull request locally
- [x] I have ran the unit tests locally
- [x] I have manually tested this pull request
- [x] I have reviewed my pull request
- [x] I have added any relevant tests

## Demonstration
This package is tested in the SDK package.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -14,20 +14,20 @@
...
 "@types/node@^17.0.34":
   version "17.0.45"
   resolved "https://registry.yarnpkg.com/@types/node/-/node-17.0.45.tgz#2c0fafd78705e7a18b7906b5201a522719dc5190"
   integrity sha512-w+tIMs3rq2afQdsPJlODhoUEKzFP1ayaoyl1CcnwtIlsVe7K7bA1NGm4s3PraqTLlXnbIN84zuBlxBWo1u9BLw==
 
-"as-proto@npm:@koinos/as-proto":
-  version "0.2.4"
-  resolved "https://registry.yarnpkg.com/@koinos/as-proto/-/as-proto-0.2.4.tgz#3ed5977578c35608e95f3459fed73b4bc416f513"
-  integrity sha512-vJnN9qQ+zbBOmPj5BREtaKwH2vS2aPFiuV4j4xPbUaaPWRU0mpPWyhYeTAsC5oh1Lf4oTNQ+y5LeUrzT/Y5aZQ==
+"as-proto@npm:@roamin/as-proto":
```
Comment (mvandeberg) [inline yarn.lock]: Once koinos/as-proto#13 is merged, this can go back to pointing to `@koinos`, correct?

Comment (roaminro) [reply yarn.lock]: The `yarn.lock` file is indeed invalid this is some leftover from my testing that I forgot to remove. Let's wait for https://github.com/koinos/as-proto/pull/13 to be merged and then I'll push the updated `yarn.lock`

**Comment by roaminro** [comment]
the `yarn.lock` file is now in sync with https://github.com/koinos/as-proto

**Comment by roaminro** [comment]
@mvandeberg @sgerbino would it be possible to review this PR? Thanks!

**Review by mvandeberg — APPROVED** [review]

**Comment by mvandeberg** [comment]
@roaminro There are now merge conflicts.

**Comment by mvandeberg** [comment]
Blocking koinos/koinos-sdk-as#71

**Comment by roaminro** [comment]
@mvandeberg I think it should be fixed now. Also, I wasn't sure about the vesion number, I'll let you edit if needed ;)

**Review by mvandeberg — APPROVED** [review]

**Pull request merged**